includes/multi-currency/Compatibility/WooCommercePointsAndRewards.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * Class WooCommercePreOrders
```

Comment:
```suggestion
 * Class WooCommercePointsAndRewards
```

Reply:
Thanks for catching it! Addressed in 34e2338